apps/dashboard/src/pages/api-keys.tsx

```diff
@@ -0,0 +1,149 @@
+      <DashboardLayout headerStartItems={<h1 className="text-foreground-950">API Keys</h1>}>
+        <div className="flex flex-col gap-6 p-2">
+          <div className="grid grid-cols-1 gap-6 pt-10 lg:grid-cols-[1fr,500px]">
+            <Form {...form}>
+              <Card className="shadow-none">
+                <CardContent className="p-6">
+                  <div className="space-y-4">
+                    <div className="space-y-2">
+                      <label className="text-foreground-600 text-sm font-medium">API Key</label>
+                      <div className="flex items-center gap-2">
+                        <InputField className="flex-1">
+                          <div className="text-foreground-600">
+                            <RiKey2Line className="size-4" />
+                          </div>
+                          <input
+                            readOnly
+                            value={showApiKey ? form.getValues('apiKey') : maskApiKey(form.getValues('apiKey'))}
+                            className="flex-1 bg-transparent outline-none"
+                          />
+                        </InputField>
+                        <Button
+                          variant="outline"
+                          size="icon"
+                          onClick={toggleApiKeyVisibility}
+                          aria-label={showApiKey ? 'Hide API Key' : 'Show API Key'}
+                        >
+                          {showApiKey ? <RiEyeOffLine className="size-4" /> : <RiEyeLine className="size-4" />}
+                        </Button>
+                        <CopyButton valueToCopy={form.getValues('apiKey')} />
+                      </div>
+                      <p className="text-foreground-600 text-xs">
+                        Use this key to authenticate your API requests. Keep it secure and never share it publicly.
+                      </p>
+                    </div>
+
+                    <div className="space-y-2">
+                      <label className="text-foreground-600 text-sm font-medium">Application Identifier</label>
+                      <div className="flex items-center gap-2">
+                        <InputField className="flex-1">
+                          <input
+                            readOnly
+                            value={form.getValues('identifier')}
+                            className="flex-1 bg-transparent outline-none"
+                          />
+                        </InputField>
+                        <CopyButton valueToCopy={form.getValues('identifier')} />
+                      </div>
+                      <p className="text-foreground-600 text-xs">
+                        The public key identifier that can be exposed to the client applications, and used to
+                        authenticate UI Components
+                      </p>
+                    </div>
+
+                    <div className="space-y-2">
```

Comment:
We discussed about this section offline and in the new Dashboard it can be removed.